### PR TITLE
Add update command

### PR DESCRIPTION
**Pull request by andesol**
## Done

- Add a `update-spreadheet` command that... updates the spreadsheet (same behaviour than the google script)
- Add configs to deploy a cronjob
- Point to another spreadsheet: [Specs tracker](https://docs.google.com/spreadsheets/d/1aKH6petyrzjzw0mgUNQscDhFSfVkbAIEjfH7YBS-bDA/edit#gid=0) (that will be the "official" if this is merged)

## QA
- `dotrun exec flask update-spreadsheet`
- Wait till the script finishes
- Check that the spreadsheet is filled
- `dotrun`
- Check that the website is not broken

## Issue
Fixes https://github.com/canonical/workplace-engineering-squad/issues/864

### CODE REVIEW

File: konf/cronjob-update.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: cronjob-update-spreadsheet
+schedule: "0 */2 * * *"
```
Comment (andesol) [inline konf/cronjob-update.yaml]: Every two hours may be enough?

File: webapp/app.py [hunk 2]
```diff
@@ -31,10 +33,11 @@
 
 def get_value_row(row, type):
     if row:
+        print("row", row)
```
Comment (tbille) [inline webapp/app.py]: :wave: print

Comment (andesol) [reply webapp/app.py]: Ops, done!

**Review by tbille — APPROVED** [review]

File: konf/cronjob-update.yaml (hunk 1 above)
Comment (tbille) [inline konf/cronjob-update.yaml]: Can you update it so it runs every hour so we keep what we had previously.

Comment (andesol) [reply konf/cronjob-update.yaml]: Done!

**Pull request merged**